travis/build-and-test.sh

```diff
@@ -27,10 +27,19 @@ OLD_CWD=$(readlink -f .)
 pushd "$BUILD_DIR"
 
 # configure build
-cmake "$REPO_ROOT"
+if [ "MEASURE_CODE_COVERAGE" == "" ]; then
+    cmake "$REPO_ROOT"
+else
+    cmake "$REPO_ROOT" -DBUILD_COVERAGE_TESTS=On
```

Comment:
The name of the CMake option is different (something overly long, I think `-DMEASURE_CODE_COVERAGE=ON`). Talking about it, can you just make it e.g., `-DTEST_COVERAGE=ON`?

Reply:
Different from ..?
This name was set attending to your recommendations.

Reply:
The name should be consistent for environment variables and CMake variables. I don't care which one really, the two in use right now are just a bit longer than needed. However, as long as they will be consistent at some point, you're free to choose whichever fits better.